daemon/cmd/daemon_main.go

```diff
@@ -629,6 +631,9 @@ func init() {
 	flags.String(option.IPv4NativeRoutingCIDR, "", "Allows to explicitly specify the IPv4 CIDR for native routing. This value corresponds to the configured cluster-cidr.")
 	option.BindEnv(option.IPv4NativeRoutingCIDR)
 
+	flags.String(option.DataPathIPv4CIDR, "", "Allows to explicitly specify the CIDR for the data path which the user desires.")
```

Comment:
Wondering out loud:
1) Does CIDR make more sense than device name in most deployments? Or maybe having both would be useful?

2) This is also very specific to tunnelling so I think it'd make sense to use a more specific parameter, e.g. `IPv4TunnelCIDR`.